csbpg/resource_binding_user.go

```diff
@@ -121,8 +152,13 @@ func resourceBindingUserDelete(ctx context.Context, d *schema.ResourceData, m an
 	if err != nil {
 		return diag.FromErr(err)
 	}
+
+	defer func(bindingUserDBConnection *sql.DB) {
+		_ = bindingUserDBConnection.Close()
+	}(bindingUserDBConnection)
+
 	log.Println("[DEBUG] reassigning object ownership")
-	_, err = bindingUserDBConnection.Exec(fmt.Sprintf("REASSIGN OWNED BY %s TO %s", pq.QuoteIdentifier(bindingUser), pq.QuoteIdentifier(cf.dataOwnerRole)))
+	_, err = bindingUserDBConnection.Exec(fmt.Sprintf("REASSIGN OWNED BY CURRENT_USER TO %s", pq.QuoteIdentifier(cf.dataOwnerRole)))
```

Comment:
Is this slightly changing the functionality? The current user executing the statement would be the admin user not the binding user. Or did I miss something?